### PR TITLE
added dtEnvitonment to player damage text

### PR DESCRIPTION
**Pull request by 12xx12**
me bad - me sleepy

### CODE REVIEW

**Comment by tigerw** [comment]
Make sure to get sufficient sleep! :)

File: src/Entities/Player.cpp [hunk 1]
```diff
@@ -1203,6 +1203,7 @@ void cPlayer::KilledBy(TakeDamageInfo & a_TDI)
 					case dtAdmin:           return "was administrator'd";
 					case dtExplosion:       return "blew up";
 					case dtAttack:          return "was attacked by thin air";
+					case dtEnvironment:     return "played to much dress up";  // This is not vanilla - added a own pun
```
Comment (tigerw) [inline src/Entities/Player.cpp:1206]: you have to explain the pun, I don't get it...

Comment (12xx12) [reply src/Entities/Player.cpp:1206]: You only get this type of damage if you are an enderman in the rain or a snowman in the desert

**Comment by 12xx12** [comment]
Eat sleep code repeat

File: src/Entities/Player.cpp (hunk 1 above)
Comment (bearbin) [reply src/Entities/Player.cpp:1206]: I prefer "succumbed to the conditions", but your choice would be fine after fixing the typo "to" -> "too".

**Review by bearbin — APPROVED** [review]

**Pull request merged**